src/components/manage/Controlpanels/Users/RenderUsers.jsx

```diff
@@ -67,7 +111,20 @@ class RenderUsers extends Component {
         <Table.Cell className="fullname">
           {this.props.user.fullname
             ? this.props.user.fullname
-            : this.props.user.username}
+            : this.props.user.username}{' '}
+          ({this.props.user.username})
+          {this.props.userschema && (
+            <>
+              <br />
+              <Button
+                onClick={() => {
+                  this.onClickEdit({ formData: this.props.user });
+                }}
+              >
+                {this.props.intl.formatMessage(messages.edit)}
```

Comment:
A personal perspective: Can we move this button inside the options/Dropdown component for individual entry? Or maybe use a "linkedButton" like plone classic UI?

Reply:
I've moved the edit button inside the actions @nileshgulia1